### DISCUSSION TITLE
Scrolling over the menu bar sliders

### DISCUSSION THREAD

**Discussion by Muirium** [MonitorControl/MonitorControl#592] [Ideas]
I use a Magic Mouse at my desk. Side scrolling over other horizontal sliders pulled down from the menubar (like system volume) adjusts those settings live, without a modal click.

Could MonitorControl behave this way, too? Scrolling sliders is second nature on the Mac!

**Comment by waydabber** [comment]
The default NSSlider does not support it but I guess it is doable. I suggest we should add this idea to the following issue:

https://github.com/MonitorControl/MonitorControl/issues/258
